thirdparty/openapi3/schemas/AuthScopesEnum.yaml

```diff
@@ -0,0 +1,9 @@
+title: AuthScopesEnum
```

Comment:
Should we change this to `AuthScopesTypes` to keep in line with mojaloop definition standard. From what I understand Enums are denoted with `...Types`.

Reply:
Yes please - if that's what we do elsewhere